server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestNodesHotThreadsAction.java

```diff
@@ -34,22 +36,42 @@
 
 import java.io.IOException;
 import java.util.List;
+import java.util.stream.Collectors;
 
 import static org.elasticsearch.rest.RestRequest.Method.GET;
 
-
 public class RestNodesHotThreadsAction extends BaseRestHandler {
 
+    private static final DeprecationLogger deprecationLogger = new DeprecationLogger(
+        LogManager.getLogger(RestNodesHotThreadsAction.class));
+    static final String DEPRECATED_MESSAGE_CLUSTER_NODES_HOTTHREADS = "[/_cluster/nodes/hotthreads] is a deprecated endpoint. " 
```

Comment:
Just question: Should we also use private here?

Consider use String.format() rather than concat string due to better performance, better readability.